DemoViperUITests/Specs/DetailRepoUISpecs.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  DetailRepoUISpecs.swift
+//  UITests
+//
+//  Created by Nguyen M. Tam on 13/01/2021.
+//  Copyright © 2021 Nimble Co. Ltd. All rights reserved.
+//
+
+import Quick
+import Nimble
+
+@testable import DemoViper
+
+final class DetailRepoUISpecs: QuickSpec {
+
+    override func spec() {
+        describe("detail repo") {
+
+            var application: XCUIApplication!
+
+            beforeEach {
+                self.setupSpec() 
+                application = setupTestEnvironment()
+                application.launch()
+                sleep(3)
+                application.tapRepoCell(at: 0)
```

Comment:
I think we can also use waitFor RepoCell to exist instead of waiting a specific time?
```
                        let table = app.tables[ListRepos.tableView.identifier]
                        let cell = table.cells.first

                        expect(table.exists &&
                                cell.exists &&
                                cell.isHittable)
                                .toEventually(beTrue(),
                                              timeout: 2.0,
                                              description: "🛑 Should have cell.")

```
We can also put it in `application.tapRepoCell` too.

Reply:
Thank you 😂 It is such a good recommendation for writing async code. Let me try.